tests/test_cli_targeted.py

```
@@ -0,0 +1,840 @@
+    @patch("mmrelay.migrate.verify_migration")
+    def test_failed_verification(self, mock_verify, mock_print):
+        """Test failed verification returns 1."""
+        mock_verify.return_value = {"ok": False}
+        mock_print.return_value = None
+
+        result = handle_verify_migration_command(self.args)
+
+        self.assertEqual(result, 1)
+
+    @patch("builtins.print")
+    def test_import_error_returns_1(self, mock_print):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_print`

(ARG002)